app/AppKernel.php

```diff
@@ -13,8 +13,11 @@ public function registerBundles()
             new Symfony\Bundle\TwigBundle\TwigBundle(),
             new Symfony\Bundle\MonologBundle\MonologBundle(),
             new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
-            new Sylius\Bundle\TaxationBundle\SyliusTaxationBundle(),
             new Sylius\Bundle\ResourceBundle\SyliusResourceBundle(),
+            new Sylius\Bundle\TaxationBundle\SyliusTaxationBundle(),
+            new Sylius\Bundle\MoneyBundle\SyliusMoneyBundle(),
```

Comment:
Multi-currency support

Reply:
Yep, but it is just there because `Sylius\Payment` needs it. 
Later, we will be able to manage currencies properly 🙂